scripts/translate_ascii.py

```diff
@@ -0,0 +1,59 @@
+import argparse
+import os
+import tempfile
+
+from scripts.aws.aws_helper import get_session, is_s3
+from scripts.files.fs import find_sidecars, write_all
+from scripts.files.fs_s3 import bucket_name_from_path, list_uri
+from scripts.gdal.gdal_helper import run_gdal
+
+
+def main() -> None:
+    """Translates ascii files in a given path to tiffs and writes to target,
+
+    Arguments:
+    --source - local or s3 path to source data
+    --target - local or s3 path to write converted tiffs
+
+    examples:
+        python translate_ascii.py --source s3://linz-elevation-staging/test/sample-tests/ --target /tmp/
+        python translate_ascii.py --source ./tests/data/ --target /tmp/
+    """
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--source", dest="source", required=True, help="S3 path to the input ascii files")
```

Comment:
I'm not sure to have a full understanding of this integrated inside a workflow.
Why not passing a path to a file (json) that contains the actual list of the ascii files. So you can exclude the logic of filtering the source directory (as we do in other scripts in this repo).

Reply:
Good point, I have changed this to from-file and undated the temporary ascii workflow to use aws-list similar to how we structured for creating thumbnails